test/regress/regress1/sets/choose3.smt2

```diff
@@ -0,0 +1,7 @@
+(set-logic ALL)
+(set-info :status unsat)
+(set-option :produce-models true)
+(declare-fun A () (Set Int))
+(assert (= (choose A) 10))
+(assert (= A (as emptyset (Set Int))))
```

Comment:
Initially I thought this benchmark should answer `sat`. Then I changed my mind guessing most users would prefer this to be `unsat` and changed the semantics accordingly. Do you agree on this behavior?

Reply:
This should be sat.  choose should have partial semantics when the set is empty.  See my other comment.

Reply:
modified to expect `sat`